src/components/modals/login/EvmProvider.tsx

```diff
@@ -0,0 +1,123 @@
+import useGetTheme from '@/hooks/useGetTheme'
+import {
+  connectorsForWallets,
+  darkTheme,
+  lightTheme,
+  RainbowKitProvider,
+} from '@rainbow-me/rainbowkit'
+import {
+  argentWallet,
+  coinbaseWallet,
+  ledgerWallet,
+  metaMaskWallet,
+} from '@rainbow-me/rainbowkit/wallets'
+import { configureChains, createConfig, WagmiConfig } from 'wagmi'
+import { mainnet } from 'wagmi/chains'
+import { publicProvider } from 'wagmi/providers/public'
+import { talismanWallet } from './wallets/talisman'
+// import { subWalletWallet } from './wallets/subWallet'
+
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet],
+  [publicProvider()]
+)
+
+const connectors = connectorsForWallets([
+  {
+    groupName: 'Recommended',
+    wallets: [
+      metaMaskWallet({ chains }),
+      talismanWallet({ chains }),
+      argentWallet({ chains }),
+      coinbaseWallet({ chains, appName: '' }),
+      ledgerWallet({ chains }),
+      // subWalletWallet({ chains }),
+    ],
+  },
+])
+
+const wagmiConfig = createConfig({
+  autoConnect: true,
+  connectors,
+  publicClient,
+  webSocketPublicClient,
+})
+
+const accentColor = '#4D46DC'
+
+const radiiConfig = {
```

Comment:
what is radii?

Reply:
This name of variable from rainbowkit theme config (https://www.rainbowkit.com/docs/custom-theme). Border radius config for modal elements